data/xml/2006.amta.xml

```diff
@@ -0,0 +1,511 @@
+      <url hash="3b3937c7">2006.amta-panels.7</url>
+    </paper>
+    <paper id="2">
+      <title>Presentation</title>
+      <author><first>Jaime</first><last>Carbonell</last></author>
+      <url hash="e1ccdfad">2006.amta-panels.9</url>
+    </paper>
+    <paper id="3">
+      <title>Presentation</title>
+      <author><first>Nizar</first><last>Habash</last></author>
+      <url hash="7e2753bb">2006.amta-panels.10</url>
+    </paper>
+    <paper id="4">
+      <title>Statistical machine translation and hybrid machine translation</title>
+      <author><first>Philipp</first><last>Koehn</last></author>
+      <url hash="d458c82c">2006.amta-panels.11</url>
+    </paper>
+    <paper id="5">
+      <title>Combining interlingua with <fixed-case>SMT</fixed-case></title>
+      <author><first>Stephanie</first><last>Seneff</last></author>
+      <url hash="4e7596e7">2006.amta-panels.12</url>
+    </paper>
+    <paper id="6">
+      <title>First strategies for integrating hybrid approaches into established systems</title>
+      <author><first>Jean</first><last>Senellart</last></author>
+      <author><first>John S.</first><last>White</last></author>
+      <url hash="03c4e9f2">2006.amta-panels.13</url>
+    </paper>
+  </volume>
+  <volume id="tutorials" ingest-date="2021-05-17">
+    <meta>
+      <booktitle>Proceedings of the 7th Conference of the Association for Machine Translation in the Americas: Tutorials</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Cambridge, Massachusetts, USA</address>
+      <month>August 8-12</month>
+      <year>2006</year>
+    </meta>
+    <paper id="1">
+      <title>A Gentle Introduction to Ontologies</title>
+      <author><first>Eduard</first><last>Hovy</last></author>
+      <url hash="0efb08d7">2006.amta-tutorials.1</url>
+    </paper>
+    <paper id="2">
+      <title>Tools and Techniques for Translatable Content</title>
+      <author><first>Mike</first><last>Dillinger</last></author>
+      <url hash="81701097">2006.amta-tutorials.2</url>
+    </paper>
+    <paper id="3">
+      <title><fixed-case>A</fixed-case>rabic Dialect Processing</title>
+      <author><first>Mona</first><last>Diab</last></author>
+      <author><first>Nizar</first><last>Habash</last></author>
+      <url hash="23be85e9">2006.amta-tutorials.3</url>
+    </paper>
+    <paper id="4">
+      <title>An Overview of Statistical Machine Translation</title>
+      <author><first>David</first><last>Smith</last></author>
+      <author><first>Charles</first><last>Schafer</last></author>
+      <url hash="90c6cd2e">2006.amta-tutorials.4</url>
+    </paper>
+    <paper id="5">
+      <title>Name Translation</title>
+      <author><first>Keith</first><last>Miller</last></author>
+      <author><first>Sherri</first><last>Condon</last></author>
+      <url hash="f5d2f989">2006.amta-tutorials.5</url>
+    </paper>
+  </volume>
+  <volume id="workshop" ingest-date="2021-05-17">
```

Comment:
I think CLAW should be its own venue. Steps to do this:

1. Add an entry to `data/yaml/venues.yaml`.
2. Ingest this as `2006.claw-1` (that's the volume ID)
3. in `data/yaml/joint.yaml`, add an entry under AMTA 2006 pointing to `2006.claw-1`. This will cause CLAW to be listed on the [AMTA 2006 event page](https://aclanthology.org/previews/ingest-amta-2006/events/amta-2006/) event though it has a separate ID.

Reply:
Right, I think this is okay now.